### PR TITLE
SOHO-7511: Comment out possibly no needed case

### PR DESCRIPTION
**Pull request by guoliang**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Fixes issue in https://jira.infor.com/browse/SOHO-7511

**Related github/jira issue (required)**:

https://jira.infor.com/browse/SOHO-7511

**Steps necessary to review your pull request (required)**:

In my case, I could not open a new tab which is hosting a an Angular component.

**Closing issues**

### CODE REVIEW

File: src/components/tabs/tabs.js [hunk 1]
```diff
@@ -2251,10 +2251,10 @@ Tabs.prototype = {
           }
           // do nothing if it's just a string of text.
           break;
-        case 'object':
-          // jQuery object or HTML Element
-          sourceString = $(sourceString).clone();
-          break;
+        // case 'object':
+        //   // jQuery object or HTML Element
```
Comment (EdwardCoyle) [inline src/components/tabs/tabs.js]: Completely remove this case instead of just commenting it out

File: src/components/tabs/tabs.js [hunk 2]
```diff
@@ -2251,10 +2251,10 @@ Tabs.prototype = {
           }
```
Comment (EdwardCoyle) [inline src/components/tabs/tabs.js]: On line 2250:
```javascript
sourceString = obj.length ? $(sourceString).clone() : sourceString;
```
Should probably also remove the clone from the ternary statement as well

Comment (guoliang) [reply src/components/tabs/tabs.js]: Fixed

File: src/components/tabs/tabs.js (hunk 1 above)
Comment (guoliang) [reply src/components/tabs/tabs.js]: Fixed

**Pull request merged**